integrations/oci/metrics/cdi/pom.xml

```diff
@@ -57,6 +62,11 @@
             <artifactId>hamcrest-all</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.mockito</groupId>
+            <artifactId>mockito-core</artifactId>
```

Comment:
I guess this is possible but another approach might be to use `java.lang.reflect.Proxy` etc. since `Monitoring` is an interface. Might be simpler.

Reply:
Good point, and I've made the change to use `java.lang.reflect.Proxy` to simplify mocking and avoid adding the Mockito dependency..